testsuite/tests/ui/auth/conftest.py

```diff
@@ -54,7 +54,12 @@ def auth0_setup(custom_admin_login, testconfig, ui_sso_integration, navigator, s
     assert sso.test_flow_checkbox.is_checked
     sso.publish()
 
-    return urls
+    yield urls
+
+    if not testconfig["skip_cleanup"]:
```

Comment:
Currently, test are not cleaning Auth0 auth source only users created by this auth source